components/encumbered/vlc/Makefile

```diff
@@ -19,6 +19,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME= vlc
 COMPONENT_VERSION= 3.0.11.1
+COMPONENT_REVISION= 1
 COMPONENT_FMRI = media/vlc
 COMPONENT_CLASSIFICATION = Applications/Sound and Video
 COMPONENT_SUMMARY= Cross-platform media player and streaming server
```

Comment:
You seem to have missed:
PKG_CONFIG_PATH+=:/usr/lib/qt/5.8/lib/$(MACH64)/pkgconfig

Reply:
and later also
QT5_ROOT = /usr/lib/qt/5.8